parm/atm/obs/config/amsua_n19.yaml

```diff
@@ -0,0 +1,323 @@
+obs space:
+  name: amsua_n19
+  obsdatain:
+    obsfile: $(OBS_DIR)/$(OBS_PREFIX).amsua_n19.$(OBS_DATE).nc4
+  obsdataout:
+    obsfile: $(DIAG_DIR)/diag_amsua_n19_$(OBS_DATE).nc4
+  simulated variables: [brightness_temperature]
+  channels: &amsua_n19_channels 1-15
+obs operator:
+  name: CRTM
+  Absorbers: [H2O,O3,CO2]
+  Clouds: [Water, Ice]
+  Cloud_Fraction: 1.0
+  obs options:
+    Sensor_ID: amsua_n19
+    EndianType: little_endian
+    CoefficientPath: $(CRTM_COEFF_DIR)
+obs bias:
+  input file: $(BIAS_DIR)/$(BIAS_PREFIX).amsua_n19.satbias.$(BIAS_DATE).nc4
+  variational bc:
+    predictors:
+    - name: constant
+    - name: lapse_rate
+      order: 2
+      tlapse: &amsua_n19_tlapse $(BIAS_DIR)/$(BIAS_PREFIX).amsua_n19.tlapse.$(BIAS_DATE).txt
+    - name: lapse_rate
+      tlapse: *amsua_n19_tlapse
+    - name: emissivity
+    - name: scan_angle
+      order: 4
+    - name: scan_angle
+      order: 3
+    - name: scan_angle
+      order: 2
+    - name: scan_angle
+obs filters:
```

Comment:
We may want to add a bias correction covariance section before obs filters.   This corresponds to the abias_pc file in the GSI.

Reply:
Here's what was added to a `amsua_n19.yaml` in a working copy of ufs_da

```
    - name: scan_angle
+  covariance:
+    minimal required obs number: 20
+    variance range: [1.0e-6, 10.0]
+    step size: 1.0e-4
+    largest analysis variance: 10000.0
+    prior:
+      input file: $(experiment_dir)/{{current_cycle}}/amsua_n19.{{previous_cycle}}.satbias_cov.nc4
+      inflation:
+        ratio: 1.1
+        ratio for small dataset: 2.0
+    output file: $(experiment_dir)/{{current_cycle}}/amsua_n19.{{current_cycle}}.satbias_cov.nc4
obs filters:
```